cdap-docs/admin-manual/source/system-requirements.rst

```diff
@@ -176,3 +179,27 @@ NTP (Network Time Protocol)
 - If you need to adjust the configuration (add or delete servers, use servers closer to you, etc.)::
 
     $ vi /etc/ntp.conf
+
+
+.. _admin-manual-cdap-and-firewalls:
+
+CDAP and Firewalls
+==================
+In your :ref:`cluster configuration <admin-manual-install-deployment-architectures>`, 
+there cannot be a firewall between the cluster and CDAP. Instead, if a
```

Comment:
Is the statement above true? This diagram in the docs seems to indicate otherwise - http://builds.cask.co/artifact/CDAP-DQB14/shared/build-1/Docs-HTML/3.5.0-SNAPSHOT/en/developers-manual/overview/modes.html